src/Session.ts

```diff
@@ -74,6 +73,25 @@ import { isSomeEnum } from "./TypeChecks.js";
 import { askColors, choosePlayer } from "./Conspiracy.js";
 import { InProduction, TestingOnly } from "./Context.js";
 
+// Tournament timer depending on the number of remaining cards in a pack.
+const TournamentTimer = [
+	0,
+	5, // 1 card remaining. Should be immediate
+	5,
```

Comment:
When testing this, I had 5s to pick the last card in a 15 cards booster pack when using tournament timer. It was not instant.

Reply:
Yeah, that's what I meant with this comment, it should be instant per the rules, but I choose to keep it at 5sec for a better user experience.